docs/packages/markdown/loader.js

```diff
@@ -157,8 +182,21 @@ module.exports = async function demoLoader() {
         return `${JSON.stringify(moduleID)}: ${getDemoIdentifier(moduleID)},`;
       })
       .join('\n')}};
+
+    export async function resolveDemoImports(requests) {
```

Comment:
@Janpot From what I understood, this function will dynamically import the modules for each demo when called (presumably in the `resolveImports` prop, like [this](https://github.com/mbrookes/material-ui/blob/3290a6beadc979db7aecb8ea0af6208d91f2443c/docs/src/modules/components/Demo.js#L258)?). 

I'm not sure what the value of the `requests` parameter will be when the function is called - is it the list of imports of that demo? If yes, should we also add a an `export const imports = ${JSON.stringify(Array.from(demoImportedModuleIDs)};`?

Reply:
This function can have two signatures. You can see here: https://jquense.github.io/jarle/#resolveImports
`requests` is just an array of module identifiers that `jarle` wants to have resolved.
The way I wrote it here relies on dynamic imports. That way it will properly code split.